ci/infra/testrunner/vars.yaml

```diff
@@ -8,6 +8,10 @@ skuba:
   srcpath: "" # path to skuba source
   verbosity: 10  # default value passed to skuba with -v
 
+#Kubectl setup
+#kubectl:
+#  path: #path to kubectl binary (defaults to /usr/bin/kubectl)
+
```

Comment:
I guess this is commented out just for documentation purposes, right? If so I am fine with it. Otherwise I'd suggest removing it.

Reply:
Yes it is documentation.